package.json

```diff
@@ -48,5 +48,28 @@
     "typescript": "4.2.3",
     "webpack": "^4.41.2",
     "webpack-merge": "^4.2.2"
+  },
+  "pnpm": {
+    "overrides": {      
+      "@angular/animations": "^16.0.0",
+      "@angular/cdk": "^16.0.0",
+      "@angular/common": "^16.0.0",
+      "@angular/core": "^16.0.0",
+      "@angular/forms": "^16.0.0",
+      "@angular/material": "^16.0.0",
+      "@angular/platform-browser": "^16.0.0",
+      "@angular/router": "^16.0.0",      
+      "@angular-devkit/build-angular": "^16.0.0",
+      "@angular-eslint/eslint-plugin": "^16.0.0",
+      "@angular-eslint/eslint-plugin-template": "^16.0.0",
+      "@angular-eslint/schematics": "^16.0.0",
+      "@angular-eslint/template-parser": "^16.0.0",
+      "@angular/compiler": "^16.0.0",
+      "@angular/compiler-cli": "^16.0.0",
+      "@angular/platform-browser-dynamic": "^16.0.0",
+      "@ngtools/webpack": "^16.0.0",
+      "ng-packagr": "^16.0.0",
+      "zone.js": "^0.13.0"
+    }
```

Comment:
Do we still need these overrides now that we removed Angular 15 compatibility?

Reply:
I'm not 100% sure. If I'm not mistaken, as the packages declare a compatibility with Angular 16 and 17 (and maybe 18 in the future), pnpm will install the highest version compatible by default, so v17. So we need the overrides to say "build with v16 even if the lib is compatible with v17".

Reply:
Can we use Angular `16` in the `devDependencies` and not `16 || 17`? As we don't bundle Angular the `peerDependencies` version is the important one for the consumer and could still be `16 || 17` even of the `devDependencies` only declare `16`.

Reply:
@sdirix thanks for your input, that's better indeed.